README.md

```diff
@@ -60,6 +62,21 @@ The successful result of a PUT or a DELETE is often not a 200 OK but a 204 No Co
 https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/200
 ```
 
+## Full-text Search
+
+The `--search` option accepts a term to full-text search. This option is available for header, method, and status references.
+
+```
+$ httpref --search clear
+Clear-Site-Data
+  Clears browsing data (e.g. cookies, storage, cache) associated with the requesting
+   website.
+205
```

Comment:
Should the `205` and `431` results contain `clear`?

Reply:
They do in their `Description`, but not in their `Name` or `Summary`.

Reply:
🤦 

Of course it is. 😅